src/containers/QuizGamePage/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import QuizGame from "../../components/QuizGame";
+import Navbar from "../../components/Navbar/index";
+
+const QuizGamePage = () => {
+  return (
+    <>
+      <Navbar />
```

Comment:
Navbar will be visible without Navbar component here.

Reply:
I will add a fix for it

Reply:
PTAL